lib/datadog/tracing/contrib/dalli/quantize.rb

```diff
@@ -8,13 +8,33 @@ module Contrib
       module Dalli
         # Quantize contains dalli-specic quantization tools.
         module Quantize
+          # BEGIN BRAZE MODIFICATION
+          # Combined regex for quantizing IDs - order matters (most specific first)
+          # GUID > BSON (24 hex) > XXHASH (16 hex) > INTEGER (2+ digits)
+          QUANTIZE_ID_REGEX = /[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}|[0-9a-f]{24}|[0-9a-f]{16}|\d{2,}/
+          # END BRAZE MODIFICATION
           module_function
 
           def format_command(operation, args)
             placeholder = "#{operation} BLOB (OMITTED)"
-            command = [operation, *args].join(' ').strip
+            # BEGIN BRAZE MODIFICATION
+            if operation == :send_multiget
+              command = [operation, *args].join(' ').strip
+            else
+              # all operations except multiget have the key as the first arg
+              command = [operation, args[0]].join(' ').strip
```

Comment:
This is a change for all operations outside of `:send_multiget`. Will this create problems?